src/es/modules/utils.js

```diff
@@ -1,5 +1,135 @@
 let validIdRegEx = /^[a-zA-Z]+[a-zA-Z0-9_-]*$/;
 
 export function isIdValid(id) {
-	return id && validIdRegEx.test(id);
+  return id && validIdRegEx.test(id);
+}
+
+export function getPixelValue(val) {
+  let valType = typeof val;
+  if (valType === 'number') { return val; }
+  if (valType === 'string') { return parseInt(val, 10); }
+  return 0;
+}
+
+/**
+ * Private function to get a single target DOM element. We will try to
+ * locate the DOM element through several ways, in the following order:
+ *
+ * 1) Passing the string into document.querySelector
+ * 2) Passing the string to jQuery, if it exists
+ * 3) Passing the string to Sizzle, if it exists
+ * 4) Calling document.getElementById if it is a plain id
+ *
+ * Default case is to assume the string is a plain id and call
+ * document.getElementById on it.
+ *
+ * @private
+ */
+function getElement(element) {
+  let result = document.getElementById(element);
+
+  //Backwards compatibility: assume the string is an id
+  if (result) {
+    return result;
+  }
+  if (typeof jQuery !== 'undefined') {
```

Comment:
Should we consider dropping jQuery/Sizzle checks here? If we want to still allow third-party query methods while remaining agnostic to libraries, we could allow for a method to be passed in as the target, or allow plugins to augment this query logic.